packages/replay/test/unit/eventBuffer/EventBufferPartitionedCompressionWorker.tes.ts

```diff
@@ -0,0 +1,161 @@
+import 'jsdom-worker';
+
+import pako from 'pako';
+
+import { BASE_TIMESTAMP } from '../..';
+import { createEventBuffer } from '../../../src/eventBuffer';
+import { EventBufferPartitionedCompressionWorker } from '../../../src/eventBuffer/EventBufferPartitionedCompressionWorker';
+import { EventBufferProxy } from '../../../src/eventBuffer/EventBufferProxy';
+
+const TEST_EVENT = { data: {}, timestamp: BASE_TIMESTAMP, type: 3 };
+
+describe('Unit | eventBuffer | EventBufferPartitionedCompressionWorker', () => {
```

Comment:
This filename is misspelled (`*.tes.ts`)

Reply:
damn, good catch! thanks 😅